lib/matplotlib/backends/backend_qt4agg.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, figure):
         if DEBUG:
             print('FigureCanvasQtAgg: ', figure)
         FigureCanvasQT.__init__(self, figure)
+        FigureCanvasQTAggBase.__init__(self, figure)
```

Comment:
and not using super here.  This was a change between qt4 and qt5 that I am not 100% sure we are handling correctly.